README.md

```diff
@@ -13,25 +13,18 @@ Technologies:
 - Express
 - React
 - Foundations CSS
```

Comment:
header of the headers above (e.g. Title:, Description:. etc) should use markdown header syntax `##`